### ISSUE TITLE
Docker connection error

### ISSUE THREAD

**Issue by AlexIoannides** [AlexIoannides/dev-setup#4]
When running the `dev-container-basic` image, executing a command  like `docker ps` results in the following error,

```text
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
```

Look into the issue and fix it!

**Comment by AlexIoannides** [comment]
See https://chatgpt.com/share/67275b09-2e78-8013-a158-57f0ebf06548

**Comment by AlexIoannides** [comment]
Fixed by #6 (because Docker was removed)
